enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/OnboardingServiceImpl.java

```diff
@@ -139,6 +133,20 @@ public OnboardingStartResponse startOnboarding(OnboardingStartRequest request) t
             process.setTimestampCreated(new Date());
         }
         process = onboardingProcessRepository.save(process);
+
+        // Check for brute force attacks
+        Calendar c = GregorianCalendar.getInstance();
```

Comment:
The one day window is a little bit hardcoded. Could we define it dynamically in similar way as other durations?

E.g. two config props
```
int maxProcessCountPerPeriod
Duration maxProcessCountPeriod
```
and have
```
...
Date timestampCheckStart = DateUtil.convertExpirationToCreatedDate(maxProcessCountPeriod)
...
```

Reply:
There was no such request for customized timeframe period checks from the client. But we can surely implement this as a separate enhancement. Feel free to raise an issue if you feel strongly about this.